src/assets/tasks.json

```diff
@@ -1,6 +1,6 @@
 [
-    {"title" : "TM-2", "description" : "Setup Angular CLI", "type" : "Story", "createdOn" : "2024-05-21", "status": "Open"},
-    {"title" : "TM-3", "description" : "Create new application", "type" : "Task", "createdOn" : "2024-05-22", "status": "Open"},
-    {"title" : "TM-4", "description" : "Task List Application - Create base", "type" : "Subtask", "createdOn" : "2024-05-23", "status": "Open"},
-    {"title" : "TM-5", "description" : "Create a “Create task“ component with add a task functionality using Angular Reactive Forms.", "type" : "Subtask", "createdOn" : "2024-05-23", "status": "In Progress"}
+    {"title" : "TM-2", "description" : "Setup Angular CLI", "type" : "STORY", "createdOn" : "2024-05-21", "status": "DONE"},
+    {"title" : "TM-3", "description" : "Create new application", "type" : "TASK", "createdOn" : "2024-05-22", "status": "OBSOLETE"},
```

Comment:
I see that here you have the key as the value for type and status but when creating a new task you set the status and type as the value of the specific enum

Reply:
see lines 59 and 61 in create task component